src/types/libdef/npm-custom/koa-helmet_v5.x.x.js

```diff
@@ -0,0 +1,153 @@
+// koa-helmet explicitely supports the same API than helmet, so I imported the
+// libdef for helmet from https://github.com/flow-typed/flow-typed/blob/master/definitions/npm/helmet_v3.x.x/flow_v0.93.x-v0.103.x/helmet_v3.x.x.js
+// Then reformatted using prettier and improved for our usage.
```

Comment:
Note: I didn't contribute this back to flow-typed because I didn't want to spend the time to write comprehensive tests for this... Please tell me if you'd prefer that I do it.